test/verify/check-packagekit

```diff
@@ -153,8 +275,8 @@ class TestUpdates(NoSubManCase):
         # no updates on the Software Updates page
         b.go("/updates")
         b.enter_page("/updates")
-        b.wait_not_present("#status")
-        b.wait_in_text("#status", "System is up to date")
+        with b.wait_timeout(300):
+            b.wait_in_text("#status", "System is up to date")
```

Comment:
Is that a related change? What suddenly takes so long here, all the package updates should be mocked and thus pkcon refresh etc. should be super fast?